components/tel/Core/Src/freertos.c

```diff
@@ -112,6 +113,7 @@ void MX_FREERTOS_Init(void) {
   /* USER CODE BEGIN Init */
 
     CAN_tasks_init();                         // Rx CAN Filter, Rx callback using CAN comms
+    CPU_LOAD_init(10, 500, &htim2);

```

Comment:
Once you flesh out your PR and rest of your code, **add defines for these numbers inside freertos.c code**.